### PR TITLE
docs(react-native): clarify 'control card' impression logging

### PR DESCRIPTION
**Pull request by jrmykolyn**
# Pull Request/Issue Resolution

#### Description of Change:

This pull request updates the React Native SDK documentation to reference and explain 'control' cards. Specifically:

- It identifies 'control' cards as a distinct type.
- It describes the unique properties of 'control' cards.
- It links out to the Android and iOS 'control' card documentation.

Myself and my team have found 'control' cards to be tricky. My hope is that this pull request makes it easier for future SDK consumers to understand and accommodate 'control' cards in their work.

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [X] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @internetisaiah as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Ana Teresa Serafino on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @internetisaiah for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@internetisaiah</td>
    <td>Developer Guide<br>Technology partners<br>Channels</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>Frontend Infrastructure & Experience (FIX)</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Comment by jrmykolyn** [comment]
Hey @internetisaiah, I have another React Native SDK pull request that's ready for review.

Thanks a lot.

File: _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Braze.requestContentCardsRefresh();
 ```
 
 {% alert important %}
-If you choose to build your own UI to display cards, you must call `logContentCardImpression` in order to receive analytics for those cards.
+If you choose to build your own UI to display cards, you must call `logContentCardImpression` in order to receive analytics for those cards. This _includes_ control cards, which must be tracked even though they are not displayed to the user.
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md:37]:
```suggestion
If you choose to build your own UI to display cards, you must call `logContentCardImpression` in order to receive analytics for those cards. This includes `control` cards, which must be tracked even though they are not displayed to the user.
```

File: _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md [hunk 2]
```diff
@@ -137,6 +141,15 @@ Classic cards have a title, description, and an optional image on the left of th
 
 For a full reference of the classic (text announcement) Content Card, see the [Android][14] and [iOS][15] documentation. For a full reference of the classic image (short news) card, see the [Android][16] and [iOS][17] documentation.
 
+### Control Content Card model properties
+
+Control cards include all of the base properties, with a few _important_ differences. Most importantly:
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md:146]:
```suggestion
Control cards include all of the base properties, with a few important differences. Most importantly:
```

File: _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md [hunk 3]
```diff
@@ -137,6 +141,15 @@ Classic cards have a title, description, and an optional image on the left of th
...
+
+- The `isControl` property is _guaranteed_ to be `true`.
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md:148]:
```suggestion
- The `isControl` property is guaranteed to be `true`.
```

File: _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md [hunk 4]
```diff
@@ -137,6 +141,15 @@ Classic cards have a title, description, and an optional image on the left of th
...
+- The `extras` property is _guaranteed_ to be empty.
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md:149]:
```suggestion
- The `extras` property is guaranteed to be empty.
```

**Review by josh-mccrowell-braze — APPROVED** [review]
Thank you for helping to improve the docs!

**Comment by jrmykolyn** [comment]
> Thank you for helping to improve the docs!

No problem, thanks for reviewing and merging the changes.

A note to my future self: don't italicize.

**Pull request merged**